llms/mlx_lm/lora.py

```diff
@@ -209,14 +223,15 @@ def run(args, training_callback: TrainingCallback = None):
     np.random.seed(args.seed)
 
     print("Loading pretrained model")
-    model, tokenizer = load(args.model)
+    model_path = get_model_path(args.model)
+    model, config, tokenizer = fetch_from_hub(model_path)
 
     # Freeze all layers
     model.freeze()
     # Convert linear layers to lora layers and unfreeze in the process
     linear_to_lora_layers(model, args.lora_layers, args.lora_parameters)
 
-    print_trainable_parameters(model)
+    print_trainable_parameters(model, config)
```

Comment:
I feel like we should get this function to work without needing to pass the config as well. It would be a more general solution.

The quantized layers have `bits` and `group_size` members, maybe we should read from that?

Reply:
Yeah, that's a much better solution. I didn't notice we had it in the QuantizedLinear layer. I will update it to use that instead.